articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+Go to the [official Node.js website](https://nodejs.org/en/) to download the Node.js version for your operating system.
+
+After installing it, open up your terminal and type in:
+
+```bash
+node –version
+```
+
+and
+
+```bash
+npm –version
+```
+
+These commands will give you the version of Node.js and node package manager respectively.
+
+Next, in your terminal type in:
+
+```bash
+npm install –global gatsby-cli
+```
+
+You just installed Gatsby and its command-line interface globally on your computer, this is the first step to interact with Gatsby using the command line interface.
+
+In order to confirm you have Gatsby installed, type in:
+
+```bash
+gatsby –version
+```
+
+### Creating a new GatsbyJS site
+
+Now let's create a static site using GatsbyJS.
+
+Open up the terminal, in your code editor if you are using [VS Code](https://code.visualstudio.com/), It comes built-in with a terminal. You will be using the GatsbyJS site template from GitHub to create your site from scratch.
+
+Create a **my-first-gatsby-site** folder, and then in your terminal, type in
+`Users/first-gatsby-site: gatsby new first-gatsby-site https://github.com/gatsbyjs/gatsby-starter-hello-world`.
+
+To see the new site, you have created, check it out using these commands.
+
+First, go into the file directory you have just created:
+
+```bash
+cd first-gatsby-site
+```
+
+Next:
+```bash
+npm run develop
+```
+
+You will then get a success message saying *“your site is running at”* :
+`http://localhost:8000`
+
+You can now open up your web browser and access `http://localhost:8000`:
+
+Open `http://localhost:8000` in your browser and you will see "hello world" displayed.
+
+Congratulations! You just built your first static site using GatsbyJS.
+
+
+Now let’s talk about your file directory, it looks overwhelming especially if you are still a beginner using [Node.js](https://nodejs.org/en/).
+
+-	**The node-modules folder:** Contains a bunch of files you may or may not need.
+-	**The public folder:** This contains all of your finished static sites that would be served on a cloud server.
+-	**The src folder:** This is the folder you will be using the most because this is where you are going to store the pages you are building for your site.
+You will find an  ***index.js*** file in the pages folder because Gatsby uses *Node.js* and *React.js* and these files will be a JavaScript file but you can also use a markdown file as well with the file extension of '.md'.
+-	**The gitignore file:** This file tells GitHub to ignore whatever file you specify in it ignore. For example API keys.
+-	**The package.json:** Contain the dependencies or packages you have installed using the node package manager *“npm install”*
+
+### Adding content to the site
+
+As excited as I am that you have created your first GatsbyJS static site, let's move forward by adding more content to the site by making it look better.
+Remember how I mentioned you will need some knowledge of React.js?
```

Comment:
Can be more compelling => How does React.js fit into building our GatsbyJS site?

Then explain!

Reply:
Done.